api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -479,6 +495,30 @@ public CohortDefinition getPatientsWhoInitiatedPreTarvDuringCurrentMonthAndScree
     return cd;
   }
 
+  /**
+   * C1: Number of patients who initiated Pre-TARV during the current month and was
+   * screened for TB C1
+   * @return
+   */
+  public CohortDefinition getPatientsWhoInitiatedPreTarvDuringCurrentMonthAndScreenedTbC1() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.setName("Patients who initiated Pre-TARV during the current month and was screened for TB C1");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+
+    CohortDefinition c1 = getPatientsWhoInitiatedPreTarvAtAfacilityDuringCurrentMonthA2();
```

Comment:
Here we also have an unused var **c1**, And also i believe that in the search where where you map the **Pop** key should be passed the *c1*. But let's maybe discuss this!

Reply:
Since **getPatientsWhoInitiatedPreTarvAtAfacilityDuringCurrentMonthC1** contains the values needed for c1, and I have used that directly, I will just remove the c1 and maintain the definition as it is